editors/code/src/main.ts

```diff
@@ -200,3 +191,34 @@ function createCommands(): Record<string, CommandFactory> {
         revealDependency: { enabled: commands.revealDependency },
     };
 }
+
+function conflictExtDetect() {
```

Comment:
small nit: i would call this function `checkConflictingExtensions()` to match how the other functions are named.